schemas/dab.draft.schema.json

```diff
@@ -376,9 +376,9 @@
               "type": "object",
               "description": "Global configuration of log level",
               "additionalProperties": false,
-              "properties": {
-                "level": {
-                  "description": "Defines logging severity levels, when in default value it will set logging level based on 'host: mode' property",
+              "patternProperties": {
+                "^[a-zA-Z_]\\w{0,}(\\.\\w{1,}){0,}$": {
+                  "description": "Defines logging severity levels for specific classes, when in null value it will set logging level based on 'host: mode' property",
```

Comment:
```suggestion
                  "description": "Defines logging severity levels for specific classes, when `null` it will set logging level based on 'host: mode' property",
```